packages/set/src/index.ts

```diff
@@ -94,7 +94,7 @@ export class ReactiveSet<T> extends Set<T> {
 export class ReactiveWeakSet<T extends object> extends WeakSet<T> {
   #triggers = new TriggerCache<T>(WeakMap);
 
-  constructor(values?: readonly T[] | null) {
+  constructor(values?: Iterable<T> | null) {
```

Comment:
https://github.com/microsoft/TypeScript/blob/main/src/lib/es2015.iterable.d.ts#LL195C47-L195C60

The above comments also apply here - in particular the `readonly` comment and the fact that TS doesn't have `WeakSetConstructor` support null or undefined, which (again) doesn't conform to [the spec](https://tc39.es/ecma262/#sec-weakset-constructor):

> 4. If iterable is either undefined or null, return set.